linux_pipeline/Jenkinsfile_developer_patch_validation

```diff
@@ -9,8 +9,8 @@ pipeline {
     parameters {
         choice(choices: 'git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git\nhttps://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git\nazure_kernel', description: 'Kernel tree repo', name: 'KERNEL_GIT_URL')
         string(defaultValue: "master", description: 'Branch to be built', name: 'KERNEL_GIT_BRANCH')
-        string(defaultValue: "", description: 'URL to raw patch file. Use space separator for multiple URLs', name: 'KERNEL_PATCH')
-        string(defaultValue: "kernel_pipeline_bvt.xml", description: 'Which tests should LISA run', name: 'LISA_TEST_XML')
+        string(defaultValue: "", description: 'Specify patch location with scp://HOSTNAME:/FULL_PATH/patch.name or URL to raw patch file (not html content). Use space separator for multiple patch files' , name: 'KERNEL_PATCH')
```

Comment:
Specify kernel patches either as scp://<hostname>:<absolute_patch_path> (ex: scp://my-scp-hostname:/home/my-username/patch.p1) or url (ex: http://my-website.com/patch.p1). Use the space separator for multiple patches.